harness/determined/pytorch/_lr_scheduler.py

```diff
@@ -75,11 +75,11 @@ def __init__(
     def step(self, *args: Any, **kwargs: Any) -> None:
         self._scheduler.step(*args, **kwargs)
 
-    def get_last_lr(self) -> List:
-        return self._scheduler.get_last_lr()
+    def get_last_lr(self) -> List[float]:
+        return list(self._scheduler.get_last_lr())
```

Comment:
I think the `list()` is not intuitive here (It only serves the purpose of having the linter not freak out. Probably better to  `type: ignore` to avoid confusion on the return type of `get_last_lr()`.